upup/models/cloudup/resources/addons/networking.weave/k8s-1.12.yaml.template

```diff
@@ -179,10 +179,13 @@ spec:
               port: 6784
           resources:
             requests:
-              cpu: 50m
-              memory: 200Mi
+              cpu: {{ or .Networking.Weave.CPURequest "50m" }}
+              memory: {{ or .Networking.Weave.MemoryRequest "200mi" }}
```

Comment:
These memory units need to have a capital `M` don't they?

https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource/#memory-units

Reply:
That's what I thought also...